api/routes/api.php

```diff
@@ -72,6 +72,7 @@
 });
 
 Route::get('/event-types/{id}/availabilities', 'Api\\EventTypeController@getAvailableTime');
+Route::get('/event-types/{id}/public/{nickname}', 'Api\\EventTypeController@getEventTypeByIdAndNickname');
```

Comment:
Can we create a separate route group for public endpoints with `public` prefix?
The idea is to have correct REST structure `api/v1/public/users/{nickname}/event-types/{eventTypeId(eventTypeSlug in a good times)}`

Reply:
Yes, if course. Good idea.